src/smc-webapp/project_store.coffee

```diff
@@ -648,24 +645,34 @@ class ProjectActions extends Actions
             opts?.finish_cb?()
         )
 
+    # Sets the active file_sort to next_column_name
+    set_sorted_file_column: (column_name) =>
+        current = @get_store()?.active_file_sort
```

Comment:
It makes no sense to put a ? here and then use `current.column_name` in the next line.  You absolutely need `current.column_name` if there is any possible reason to use `@get_store()?`.